supp2u/src/BasicRoute.js

```diff
@@ -1,41 +1,48 @@
+import EditReviewPresentation from "./components/Reviews/edit";
+import BusinessSingleView from "./components/Business/BusinessSingleView";
 import AddCustomer from "./components/Customer/Add";
 import CustomerView from "./components/customerviews/CustomerView";
-import InitLandingPage from "./components/InitLandingPage"
+import InitLandingPage from "./components/InitLandingPage";
 import BusinessUpdater from "./components/Business/BusinessUpdater";
-import Map from './components/googleMap/Map'
+import Map from "./components/googleMap/Map";
 
+export default () => {
+  return (
+    <>
+      {/* Special Auth0 Route  */}
+      <Route path={"/login"} exact component={Auth0} />
 
-export default () =>{
-  return <>
-        {/* Main Page */}
-        <Route path={"/home"} component={Map}/>
-        {/* Special Auth0 Route  */}
-        <Route path={"/login"} exact component={Auth0}/>
-        {/* Business */}
-        <Route path={"/businesses/create"} exact component={BusinessCreator}/>
-        <Route path={"/business/:id"} exact component={BusinessSingleView} />
-        <Route path={"/business/update/:id"} exact component={BusinessUpdater}/>
+      {/* Business */}
+      <Route path={"/businesses/create"} exact component={BusinessCreator} />
+      <Route path={"/business/:id"} exact component={BusinessSingleView} />
+      <Route path={"/business/update/:id"} exact component={BusinessUpdater} />
+      <Route path={"/review/:id"} exact component={EditReviewPresentation} />
+      
+      {/* Customer */}
+      <Route path={"/register/customer"} component={AddCustomer} />
+      <Route path={"/customer/create"} exact component={CustomerCreator} />{" "}
+      {/* have a single way of creating customer */}
+      <Route path={"/customer/view"} exact component={CustomerView} />{" "}
+      {/* will break unless you have a customer_id */}
 
-        <Route path={"/review/:id"} exact component={EditReviewPresentation} />
+      {/* Menu */}
+      <Route path={"/menu/new"} exact component={MenuCreator} />
+      <Route path={"/schedule/create"} exact component={ScheduleCreator} />
 
-        {/* Customer */}
-        <Route path={"/register/customer"} component={AddCustomer}/>
-        <Route path={"/customer/create"} exact component={CustomerCreator}/> {/* have a single way of creating customer */}
-        <Route path={"/customer/view"} exact component={CustomerView}/> {/* will break unless you have a customer_id */}
-        {/* Menu */}
-        <Route path={"/menu/new"} exact component={MenuCreator} />
-        <Route path={"/schedule/create"} exact component={ScheduleCreator}/>
-        {/* registration */}
-        <Route path={"/registration"} exact component={InitLandingPage} />
-      </>
-}
\ No newline at end of file
+      {/* registration */}
+      <Route path={"/registration"} exact component={InitLandingPage} />
+
+      {/* Main Page */}
+      <Route exact path={"/"} component={Map} />
```

Comment:
If map is rendered at path="/" Then it will render on every page with every component, we do need to find a way to make the map load on the / (maybe redirect to /home), but doing this will cause the map to render everywhere

Reply:
If you want to you can make a new componet that renders at "/" then redirects them to /home

Reply:
it does not

Reply:
it uses exact path and so it will not show up everytime